shell/platform/android/test/io/flutter/plugin/common/StandardMethodCodecTest.java

```diff
@@ -0,0 +1,90 @@
+package io.flutter.plugin.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@Config(manifest = Config.NONE)
+@RunWith(RobolectricTestRunner.class)
+public class StandardMethodCodecTest {
+
+  @Test
+  public void encodeMethodTest() {
+    Map<String, String> args = new HashMap<>();
+    args.put("testArg", "testValue");
+    MethodCall call = new MethodCall("testMethod", args);
```

Comment:
final these (to make sure we don't mess it up in the future since you're actually asserting them later)